pyanaconda/display.py

```diff
@@ -50,6 +50,17 @@
 stdout_log = get_stdout_logger()
 
 
+X_FAILURE_MESSAGE = \
+    "X startup failed, falling back to text mode. There are multiple ways to help this issue:\n" \
+    "* Do not load the stage2 image over a slow network link.\n" \
+    "* Wait for the X server startup longer with the inst.xtimeout=<SECONDS> boot option. " \
+    "The default is 60 seconds.\n" \
```

Comment:
Could add two spaces before the line, so it has the same indentation as the one above?

Reply:
I'm afraid you expect too much :) It's literally just bullet points that get wrapped by default, no padding, nothing. But... challenge accepted!

Reply:
Oh, I thought that each line ends with `\n`. Never mind.